DQM/BeamMonitor/plugins/OnlineBeamMonitor.cc

```diff
@@ -28,13 +28,14 @@ using namespace reco;
 //----------------------------------------------------------------------------------------------------------------------
 OnlineBeamMonitor::OnlineBeamMonitor(const ParameterSet& ps)
     : monitorName_(ps.getUntrackedParameter<string>("MonitorName")),
-      bsTransientToken_(esConsumes<edm::Transition::BeginLuminosityBlock>()),
-      bsHLTToken_(esConsumes<edm::Transition::BeginLuminosityBlock>()),
-      bsLegacyToken_(esConsumes<edm::Transition::BeginLuminosityBlock>()),
+      bsOnlineToken_(consumes(ps.getUntrackedParameter<edm::InputTag>("OnlineBeamSpotLabel"))),
+      bsHLTToken_(esConsumes<edm::Transition::Event>()),
+      bsLegacyToken_(esConsumes<edm::Transition::Event>()),
```

Comment:
I think `<edm::Transition::Event>` is the default, and maybe this could be simplified to
```suggestion
      bsHLTToken_(esConsumes()),
      bsLegacyToken_(esConsumes()),
```
?